src/App.jsx

```diff
@@ -190,14 +154,23 @@ function App() {
 	if (errorState) {
 		return <ErrorMessage />;
 	}
-
+	function handleChildTabEvent(data) {
+		setSuggValidatorsData ({...data});
+		let individualStake = parseFloat(suggValidatorsData.budget)/16;
+		validators = data.validatorsList.reduce((acc, cur) => {
```

Comment:
Why is `validators` being re-assigned? Why not use react's setState when using it in the state?

Reply:
I don't think we can apply reduce function inside the setState. In order to update the validator, the validator is set in next line.